lib/html_parser.dart

```diff
@@ -434,10 +435,15 @@ class HtmlParser extends StatelessWidget {
       );
     } else {
       ///[tree] is an inline element.
-      return TextSpan(
```

Comment:
This is breaking as you are not staying in 'span-land' but are using a widget, which potentially breaks line-wrapping.

Reply:
Yep I considered this. When I made this change the example app rendered fine so I decided to go ahead and push it. 

This was necessary for supporting elements like `<span>`, `<code>`, `<q>`, etc because they don't meet any of the other criteria for other elements in the `parseTree` function. 

The tags like `<code>` still displayed inline with this change. Do you know of a good way to test the inline flow? I know you worked with this type of issue with the links, so maybe a similar test process could help determine whether this is indeed an issue.

Reply:
Let me give it a try.

Reply:
Yeah this is indeed an issue as I suspected. This is how it should look like:
![Screenshot 2021-04-29 at 10 47 06](https://user-images.githubusercontent.com/196453/116524918-3e815e00-a8d8-11eb-9e18-d1cb9d3e82c3.png)

But with your code it looks like:
![Screenshot 2021-04-29 at 10 47 44](https://user-images.githubusercontent.com/196453/116524994-55c04b80-a8d8-11eb-957c-cde4f81005df.png)

As the `<span>` I used (to make the background yellow) is now a separate widget instead of an inline span.

I revered this change but left your other contributions.

I used this test html:
```
<p>Lorem ipsum dolor amet, <span>consectetur adipiscing</span> elit, <a href="#bottom"><span>sed do eiusmod tempor</span> incididunt</a> ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
```